pymc3/distributions/continuous.py

```diff
@@ -3786,6 +3759,31 @@ def logp(value, mu, sigma):
         )
 
 
+class InterpolatedRV(RandomVariable):
+    name = "interpolated"
+    ndim_supp = 0
+    ndims_params = [0, 0]
+    dtype = "floatX"
+    _print_name = ("Interpolated", "\\operatorname{Interpolated}")
+
+    @classmethod
+    def rng_fn(cls, rng, p, pdf, cdf, x, size=None):
```

Comment:
Something is of with the `rng_fn` signature. `p` seems to be overwritten so it shouldn't be there, and if you want to pass the `cdf` already specified it should also be in the `dist` classmethod bellow and `ndim_params` shoud be 3 elements long